sormas-backend/src/main/java/de/symeda/sormas/backend/campaign/CampaignFacadeEjb.java

```diff
@@ -428,7 +428,9 @@ public List<ProcessedEntity> archive(List<String> entityUuids) {
 	}
 
 	@Override
-	@RightsAllowed(UserRight._CAMPAIGN_ARCHIVE)
+	@RightsAllowed({
```

Comment:
@RightsAllowed allows access if the user has one of the given rights so a user with view archived right will be allowed to dearchive a campaign

The view archive right must be checked in the method

Please change this for all the dearchive methods